rotkehlchen/tests/api/test_data_purging.py

```diff
@@ -239,29 +244,40 @@ def _assert_zksynclite_txs_num(cursor: 'DBCursor', tx_num: int, swap_num: int) -
             address=BTCAddress('bc1xxxxxxxxxx'),
         )
 
+    # check deleting by hash
     response = requests.delete(
-        api_url_for(
-            rotkehlchen_api_server,
-            'blockchaintransactionsresource',
-        ),
+        api_url_for(rotkehlchen_api_server, 'blockchaintransactionsresource'),
+        json={'chain': 'btc', 'tx_hash': event_identifier2.removeprefix(BTC_EVENT_IDENTIFIER_PREFIX)},  # noqa: E501
```

Comment:
here for example 

```suggestion
        json={'chain': 'btc', 'tx_hash': BTC_TX_HASH1,  # noqa: E501
```

Otherwise it's really hard to understand what you do in the test